util/client.go

```diff
@@ -181,6 +195,10 @@ func (c *Client) lazyInit() error {
 	childToken := childTokenLease.Auth.ClientToken
 	policies := childTokenLease.Auth.Policies
 
+	// store the expiration time of the token.
+	expireTime = expireTime.Add(time.Duration(childTokenLease.Auth.LeaseDuration) * time.Second)
```

Comment:
```suggestion
	expireTime = time.Now().Add(time.Duration(childTokenLease.Auth.LeaseDuration) * time.Second)
```
What's the point to initialize it above?

Reply:
Otherwise it may be expired before the this expireTime, I get it.